src/algorithms/tonal/audio2pitch.cpp

```diff
@@ -112,7 +112,7 @@ void Audio2Pitch::compute() {
     _spectrum->compute();
     _pitchAlgorithm->input("spectrum").set(spectrum);
   }
-  else {
+  else if (_pitchAlgorithmName == "pyin") {
```

Comment:
"pyin" is not a good name, because this abbreviation stands for pYIN, probabilistic YIN

I propose using `pitchyin`and `pitchyinfft` which directly correspond to our algorithm names PitchYin and PitchYinFFT. 

In the algorithm's DOC description, Mention these algorithms' names explicitly (e.g., "The algorithm internally uses pitch estimation with PitchYin (`pitchyin`) and PitchYinFFT (`pitchyinfft`)".

This way the documentation script will automatically add PitchYin and PitchYinFFT as related algorithms.

Reply:
alright, I will change it.